### ISSUE TITLE
Issue on download playlist 

### ISSUE THREAD

**Issue by lucasdavid47** [LatentFreedom/youtube-downloader#2]
Hi, 
I tried to download a playlist ~50 songs.
The tracks appear well separated in the folder. 
In reality all the files are identical and correspond (the audio) to the first music of the playlist.

![image](https://user-images.githubusercontent.com/33765469/166145219-e2dc7d0c-7ff7-4941-9834-57cd93edbc7d.png)

**Comment by Nevish-302** [comment]
can i try to solve this issue?

**Comment by lucasdavid47** [comment]
That would be nice of you @Nevish-302

**Comment by Nevish-302** [comment]
when i try testing the app, it shows could not download "Song Name"
NoneType' object has no attribute 'span'
why is this happening

**Comment by lucasdavid47** [comment]
Did you do every steps (README file)

**Comment by Nevish-302** [comment]
yes

**Comment by LatentFreedom** [comment]
I have fixed a bug int he code for the audio rewriting the same song over and over. Note that there is a problem within the pytube library. I have updated requirements.txt to include the most recent pytube package but the problem still seems to be there. The fix I found is updating some code within the pytube ciphers.py file. check out this link [https://www.codegrepper.com](https://www.codegrepper.com/code-examples/python/pytube.exceptions.RegexMatchError%3A+get_throttling_function_name%3A+could+not+find+match+for+multiple)
